### ISSUE TITLE
Channels with long names mangle display of Video Description

### ISSUE THREAD

**Issue by atomjack** [ram-on/SkyTube#212]
Channels whose names are really long end up mangling the display of the Video Description layer that you drag up while watching a video. An example of this is "The Late Show with Stephen Colbert":

![screenshot_20180102-115108](https://user-images.githubusercontent.com/114672/34497701-7201df66-efb3-11e7-888e-0a4827efddd9.png)

Thoughts, @ram-on ?

**Comment by ram-on** [comment]
Good catch.

**Comment by atomjack** [comment]
I came up with a quick fix:

![screenshot_20180102-122921](https://user-images.githubusercontent.com/114672/34498941-a3f6d8a0-efb8-11e7-986b-aa7854ec00b5.png)

I put values in for `weightSum` on the parent container of the two RelativeLayouts, and `layout_weight` on the RelativeLayouts. Strangely, using `weightSum` of 2 and `layout_weight` of 1 for both RelativeLayouts didn't work - I ended up having to use a `layout_weight` of 2.2 for the first one in order for it to look ok (a value of 2 caused the Unsubscribe button text to leave the last letter, e, on a new line).

It doesn't appear to change anything when the Channel Name is shorter.

Lemme know if you want me to push this up, @ram-on.
